### PR TITLE
Integrating CLI metrics with Meroxa API

### PR DESCRIPTION
**Pull request by raulb**
# Description of change

This pull-request is to accommodate the necessary changes for https://github.com/meroxa/platform-api/issues/432.

# Type of change

- [ ]  New feature
- [x]  Bug fix
- [x]  Refactor
- [ ]  Documentation

# How was this tested?

- [x]  Unit Tests
- [x]  Tested with local API

### CODE REVIEW

File: cmd/meroxa/global/metrics.go [hunk 1]
```diff
@@ -46,6 +46,10 @@ func NewPublisher() cased.Publisher {
 	if v := Config.GetBool("CASED_DEBUG"); v {
 		options = append(options, cased.WithDebug(v))
 	}
+
+	// TODO: Re-use HTTP Transform from our own Meroxa client
```
Comment (dianadoherty) [inline cmd/meroxa/global/metrics.go:50]: Do you have a ticket to address this TODO?

Comment (raulb) [reply cmd/meroxa/global/metrics.go:50]: @dianadoherty Not yet! thanks for the call out.

Comment (raulb) [reply cmd/meroxa/global/metrics.go:50]: @dianadoherty https://github.com/meroxa/meroxa-go/issues/46

**Review by dianadoherty — APPROVED** [review]

File: cmd/meroxa/global/metrics.go (hunk 1 above)
Comment (raulb) [reply cmd/meroxa/global/metrics.go:50]: Update: Right issue is https://github.com/meroxa/product/issues/16.

**Pull request merged**